### ISSUE TITLE
Changed the net and can get higher acc,but eval precision becomes lower?

### ISSUE THREAD

**Issue by 714333462** [nwojke/cosine_metric_learning#34]
i changed the net and the out feature dimension is 256, and i trained about 8hours,when i eval the model the precision 1 , 5, 10, 20 is lower? the precision 1 is only 0.66
and i find that the same net trained 2 hours, the precision 1 is 0.77, why the model trained longer get the precision lower?

**Comment by nwojke** [comment]
That sounds like overfitting and you probably need to change hyperparameters. Maybe it is a good idea to start with the weight decay of the scale parameter. It is currently hard-coded [here](https://github.com/nwojke/cosine_metric_learning/blob/master/nets/deep_sort/network_definition.py#L95). Does it help to increase this value?
